darkdetect/base.py

```diff
@@ -0,0 +1,89 @@
+from typing import Callable
```

Comment:
In general I would rename this module to be a) private, b) more explicit, so something like `_base_listener`.